_posts/0000-01-02-Mshadab123.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
# this is bullshit
## why I'm doing this
###### okay okay ill get it sooner or later

Reply:
Yeah you will get it soon